features/step_definitions/purchase_steps.rb

```diff
@@ -22,9 +22,9 @@
 
 Then(/^the book is added to my cart$/) do
   @cart = @user.cart
-  expect(@cart.order_items).to_not eq(nil)
-  last_order_item = @cart.order_items.last
-  expect(last_order_item.book).to eq(@book)
+  expect(@cart.line_items).to_not eq(nil)
+  last_line_item = @cart.line_items.last
+  expect(last_line_item.book).to eq(@book)
```

Comment:
I'd rather see this written entirely from the user's perspective. So, instead of doing `@user.cart` (which is leveraging the BE to get what's in the DB), you should pass your expectations in. So, here that would mean doing:

````
When I visit my cart
Then I see "10,000 Leagues Under the Sea" in my cart
````